src/main/java/duke/command/EventCommand.java

```diff
@@ -0,0 +1,48 @@
+package duke.command;
+
+import duke.main.Storage;
+import duke.main.TaskList;
+import duke.main.Ui;
+import duke.task.Event;
+
+import duke.exception.DukeException;
+
+/**
+ * Represents a command to add an event task to the task list.
+ */
+public class EventCommand extends Command {
+    private String description;
+    private String startDateTimeStr;
+    private String endDateTimeStr;
```

Comment:
Minor nitpick: Not necessary to include type in variable names in a strongly typed language such as Java